<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ahm-l-amitg" failures="1" tests="1" name="Test1" time="25.693" errors="0" timestamp="2023-03-18T21:10:53 IST">
  <testcase classname="com.screenshot.result.TestFaceBookTest" name="LoginFaceBook" time="17.823">
    <failure type="java.lang.AssertionError" message="expected [1] but found [0]">
      <![CDATA[java.lang.AssertionError: expected [1] but found [0]
at com.screenshot.result.TestFaceBookTest.LoginFaceBook(TestFaceBookTest.java:37)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- LoginFaceBook -->
</testsuite> <!-- Test1 -->
